common/js/src/popup-window/popup-opener.js

```diff
@@ -98,9 +100,7 @@ GSC.PopupOpener.createWindow = function(url, windowOptions, opt_data) {
             'frame': windowOptions['frame'],
             'hidden': windowOptions['hidden'],
             'id': windowOptions['id'],
-            'innerBounds': {
```

Comment:
Is this change intentional? I suppose the old code was actually correct, since `chrome.app.window.create()` takes the "innerBounds" parameter.